nixos/modules/services/networking/adguardhome.nix

```diff
@@ -135,6 +149,10 @@ in {
       };
     };
 
-    networking.firewall.allowedTCPPorts = mkIf cfg.openFirewall [ cfg.port ];
+    networking.firewall.allowedTCPPorts = mkIf cfg.openFirewall [ cfg.settings.bind_port ];
+  };
+
+  meta = {
+    buildDocsInSandbox = false;
   };
```

Comment:
Can we use defaultText to avoid this?

Reply:
I have switched to `defaultText` with the previously mentioned disadvantage, but added a note in the description of the option. `buildDocsInSandbox` has been dropped.

I think that this poses a somewhat suitable compromise.